crates/core/src/surfnet/svm.rs

```diff
@@ -323,7 +326,10 @@ impl SurfnetSvm {
         self.account_registry
             .entry(account.owner)
             .or_insert_with(Vec::new)
-            .push((*pubkey, account));
+            .push((*pubkey, account.clone()));
+
+        // Notify account subscribers
+        self.notify_account_subscribers(pubkey, &account);
```

Comment:
Teeny tiny optimization, not that it really matters.

But would it work if you moved this notify to _before_ the `self.account_registry` to remove the need to do a clone when pushing to the account registry?